Source/SharpVectorCore/Utils/PathUtils.cs

```diff
@@ -61,10 +61,19 @@ private static string GetAssemblyPathInternal(Assembly assembly, string location
 			if (!string.IsNullOrEmpty(location))
 				return location;
 
-			var baseDirectory = AppDomain.CurrentDomain.BaseDirectory;
+			var baseDirectory = GetBaseDirectory();
 			var assemblyName = GetAssemblyFileName(assembly);
 
 			return Path.Combine(baseDirectory, assemblyName);
 		}
+
+		private static string GetBaseDirectory()
+		{
+#if NET5_0
```

Comment:
Please use the project defiled constants.

Reply:
replaced `NET5_0` with `NET50` (already defined in the project file)

Reply:
Shouldn't it be NETCORE? AppDomains haven't been a thing since .NET Core 1.0

Reply:
the project does not target "netcoreapp" directly
https://github.com/ElinamLLC/SharpVectors/blob/e46133fa125ea35160d746f5e08fb366078799ff/Source/SharpVectorCore/SharpVectors.Core.csproj#L3

So I am not sure how it will be resolved. However, in the project `NETCORE` is defined for `net5` as well, so I think it is better to use NETCORE (maybe in the future there the project will reference *netcoreapp* directly)